addons/event/models/event_mail.py

```diff
@@ -47,14 +47,19 @@ class EventMail(models.Model):
         # event based: end date
         ('after_event', 'After the event ended'),
         ('before_event_end', 'Before the event ends')],
-        string='Trigger ', default="before_event", required=True)
+        string='Trigger ', default="before_event", required=True,
+        help="Indicates when the communication is sent."
+        "If the event has multiple slots, the interval is related to each time slot instead of the whole event.")
```

Comment:
```suggestion
        help="Indicates when the communication is sent. "
        "If the event has multiple slots, the interval is related to each time slot instead of the whole event.")
```
:disguised_face: I'm helping